cfme/tests/automate/custom_button/test_service_vm_custom_button.py

```diff
@@ -137,8 +136,7 @@ def test_custom_button_display_service_vm(request, appliance, service_vm, button
 
 @test_requirements.customer_stories
 @pytest.mark.tier(1)
-# Dynamic dialog problem 1729594
-@pytest.mark.meta(automates=[1687061], blockers=[BZ(1729594), BZ(1729046)])
```

Comment:
@digitronik shouldn't BZ(1729594), BZ(1729046) be also moved to automates?

Reply:
@lina-nikiforova actually these BZ are blocker to tests not part of test. When BZ effect on `test steps` or `intention/ purpose of test` then we can include in `automates` meta.
BZ affect dynamic dialog (overall); I think, we have separate test case for it.